scripts/build.py

```diff
@@ -174,9 +174,12 @@ def main():
   parser.add_argument('--skip_tests', default=False, action='store_true',
       help='Skip unit tests.')
   opts=parser.parse_args()
+  if not os.path.exists(os.path.expanduser(opts.ssl_root)):
```

Comment:
for iOS are you expecting openssl always used?

Reply:
It could be empty, which means BoringSSL will be used. I'll allow it to be empty.